README.md

```diff
@@ -40,16 +40,24 @@ Future development of the timeline component might include new features such as
 As with any design project, the timeline would also benefit hugely from user feedback. A specific concern that would be worth exploring is whether or not the zoom functionality is intuitive to users when looking at the timeline; if not, it may be beneficial to add in manual zoom buttons.
 
 ## Bar chart
-
-The bar chart is a visulization element that helps with the second major question when the number of pushes is not very large. It features a boxplot that allows user to compare the current push to the previous ones. It shows a default number of most recent pushes when the user enters the page. 
-
-![Bar Chart](/images/barchart-static.png?raw=true "Bar Chart")
-
-The bar chart includes several interactive features: a selector to slide and select any portion to display, a drop down menu to switch between all pushes and completed pushes, and a tooltip that shows push information when the user hovers on a bar or the empty area above it.
-
+ 
+### Demo
+ 
+![Bar Chart](/images/barchart.gif?raw=true "GIF of Bar Chart")
+ 
+### Description
+ 
+The bar chart is a visualization element that helps with the second major question when the number of pushes is not very large. It displays the pushes in chronological order, because the pushes happened most recently is probably more important to the user. The _y_ axis of the bar chart is the duration of a single push, with the most common time unit of the pushes. The bar chart consists of a focus bar chart, a compressed bar chart with a selector, and a boxplot. It shows a default number of pushes when the user opens the page.
```

Comment:
s/second major question when/second major question: when/
s/is probably/are probably/

I think the last sentence sounds incomplete because it's not clear what 'It' talks about.

Also note that you are using both 'bar chart' and barchart'. It would be good to stick with one form. Same for 'box plot' and 'boxplot'.